php/class-photonfill.php

```diff
@@ -1001,7 +1001,7 @@ public function get_lazyload_image( $attachment_id, $size = 'full', $attr = arra
 		 * @param array  $attr Image attributes.
 		 * @return string Picture markup.
 		 */
-		public function get_attachment_picture( $attachment_id, $size = 'full', $attr ) {
+		public function get_attachment_picture( $attachment_id, $size = 'full', $attr = array() ) {
```

Comment:
Does this project not use the short array syntax?

Reply:
It does not, unfortunately.